
int8  radio_id
int8  n_connected_devs
int16 signal
bool  ssh_active
